src/Users/Users/examples/v1.0-beta/Get-MgUser.md

```diff
@@ -16,6 +16,12 @@ UserPrincipalName : AdeleV@contoso.com
 
 This command returns a list of all users.
 
+One of the following permissions is required to run this cmdlet;
+|Permission type| Permissions (from least to most privileged)
+|-----|------|
+|Delegated (work or school account)|User.ReadBasic.All, User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All, Directory.AccessAsUser.All|
+|Application|User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All|
```

Comment:
@davidmu1, I have added permissions for this cmdlet for both v1.0 and beta versions. I have adopted the version used in the API ref content. Let me know your thoughts.